examples/window.rs

```diff
@@ -670,9 +695,58 @@ impl WindowState {
         self.window.set_cursor(cursor);
     }
 
+    /// Custom cursor from an URL.
+    #[cfg(web_platform)]
+    fn url_custom_cursor(&mut self, event_loop: &ActiveEventLoop) {
+        use winit::platform::web::CustomCursorExtWebSys;
+
+        let cursor = CustomCursor::from_url(
+            format!(
+                "https://picsum.photos/128?random={}",
+                URL_COUNTER.fetch_add(1, Ordering::Relaxed)
+            ),
+            64,
+            64,
+        );
```

Comment:
That thing looks like duplicated in the `animation_cursor` as well, so should likely extract it.

Reply:
I mean, having a function that returns you the thing is not a big of a deal, so that way the URL stays the same for both places for sure. It's less code in the end.